src/tools/miri/tests/fail/const-ub-checks.stderr

```diff
@@ -1,9 +1,24 @@
+<<<<<<< HEAD:src/tools/miri/tests/fail/const-ub-checks.stderr
 error[E0080]: evaluation of constant value failed
   --> $DIR/const-ub-checks.rs:LL:CC
+||||||| constructed merge base:src/test/ui/consts/const_limit/const_eval_limit_reached.stderr
+error[E0080]: evaluation of constant value failed
+  --> $DIR/const_eval_limit_reached.rs:6:11
+=======
+warning: constant evaluation is taking a long time
+  --> $DIR/const_eval_limit_reached.rs:6:1
+>>>>>>> Remove const eval limit and implement an exponential backoff lint instead:src/test/ui/consts/const_limit/const_eval_limit_reached.stderr
    |
+<<<<<<< HEAD:src/tools/miri/tests/fail/const-ub-checks.stderr
 LL |     ptr.read();
    |     ^^^^^^^^^^ accessing memory with alignment ALIGN, but alignment ALIGN is required
+||||||| constructed merge base:src/test/ui/consts/const_limit/const_eval_limit_reached.stderr
+LL |     while x != 1000 {
+   |           ^^^^^^^^^ exceeded interpreter step limit (see `#[const_eval_limit]`)
+=======
+LL | const X: usize = {
+   | ^^^^^^^^^^^^^^
+>>>>>>> Remove const eval limit and implement an exponential backoff lint instead:src/test/ui/consts/const_limit/const_eval_limit_reached.stderr
```

Comment:
Is this a rebase artifact?

Reply:
yea, miri testing was broken and just fixed in the last few hours, so tests didn't run at all.